{"priority": "major", "kind": "bug", "repository": {"links": {"self": {"href": "data/repositories/osrf/release-tools.json"}, "html": {"href": "#!/osrf/release-tools"}, "avatar": {"href": "data/bytebucket.org/ravatar/{41811d3d-19d0-428e-a134-976595e72508}ts=python"}}, "type": "repository", "name": "release-tools", "full_name": "osrf/release-tools", "uuid": "{41811d3d-19d0-428e-a134-976595e72508}"}, "links": {"attachments": {"href": "data/repositories/osrf/release-tools/issues/100/attachments_page=1.json"}, "self": {"href": "data/repositories/osrf/release-tools/issues/100.json"}, "watch": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/release-tools/issues/100/watch"}, "comments": {"href": "data/repositories/osrf/release-tools/issues/100/comments_page=1.json"}, "html": {"href": "#!/osrf/release-tools/issues/100/build-cop-sdformat-trusty-failed-to-fetch"}, "vote": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/release-tools/issues/100/vote"}}, "reporter": {"display_name": "Shane Loretz", "uuid": "{656e3311-aad9-45a1-aaf7-b0ee0e84b287}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B656e3311-aad9-45a1-aaf7-b0ee0e84b287%7D"}, "html": {"href": "https://bitbucket.org/%7B656e3311-aad9-45a1-aaf7-b0ee0e84b287%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:684383ab-ac95-4859-a350-4a6f41a94a22/c7a1ebf5-cade-4115-9f26-9d3facb776db/128"}}, "nickname": "Shane Loretz", "type": "user", "account_id": "557058:684383ab-ac95-4859-a350-4a6f41a94a22"}, "title": "[build cop] sdformat trusty failed to fetch package from archive", "component": null, "votes": 0, "watches": 1, "content": {"raw": "Copied from https://bitbucket.org/osrf/gazebo/issues/2184/build-cop-sdformat-trusty-failed-to-featch\r\n\r\nhttp://build.osrfoundation.org/view/main/view/BuildCopTests/job/sdformat-ci-default-trusty-amd64/46/\r\n\r\n```\r\nErr http://archive.ubuntu.com/ubuntu/ trusty-security/main linux-libc-dev amd64 3.13.0-107.154\r\n  404  Not Found [IP: 91.189.88.161 80]\r\nFetched 101 MB in 3min 5s (547 kB/s)\r\nW: Duplicate sources.list entry http://archive.ubuntu.com/ubuntu/ trusty-updates/main amd64 Packages (/var/lib/apt/lists/archive.ubuntu.com_ubuntu_dists_trusty-updates_main_binary-amd64_Packages.gz)\r\nW: Duplicate sources.list entry http://archive.ubuntu.com/ubuntu/ trusty-updates/restricted amd64 Packages (/var/lib/apt/lists/archive.ubuntu.com_ubuntu_dists_trusty-updates_restricted_binary-amd64_Packages.gz)\r\nW: Duplicate sources.list entry http://archive.ubuntu.com/ubuntu/ trusty-updates/universe amd64 Packages (/var/lib/apt/lists/archive.ubuntu.com_ubuntu_dists_trusty-updates_universe_binary-amd64_Packages.gz)\r\nW: You may want to run apt-get update to correct these problems\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/l/linux/linux-libc-dev_3.13.0-107.154_amd64.deb  404  Not Found [IP: 91.189.88.161 80]\r\n\r\nE: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?\r\n```\r\n\r\nLooks like the docker layer that installs packages was separate from the layer that updates the package list. I'm testing a fix in http://build.osrfoundation.org/view/main/view/BuildCopTests/job/sdformat-ci-default-trusty-amd64/48/", "markup": "markdown", "html": "<p>Copied from <a href=\"https://bitbucket.org/osrf/gazebo/issues/2184/build-cop-sdformat-trusty-failed-to-featch\" rel=\"nofollow\" class=\"ap-connect-link\">https://bitbucket.org/osrf/gazebo/issues/2184/build-cop-sdformat-trusty-failed-to-featch</a></p>\n<p><a href=\"http://build.osrfoundation.org/view/main/view/BuildCopTests/job/sdformat-ci-default-trusty-amd64/46/\" rel=\"nofollow\" class=\"ap-connect-link\">http://build.osrfoundation.org/view/main/view/BuildCopTests/job/sdformat-ci-default-trusty-amd64/46/</a></p>\n<div class=\"codehilite\"><pre><span></span>Err http://archive.ubuntu.com/ubuntu/ trusty-security/main linux-libc-dev amd64 3.13.0-107.154\n  404  Not Found [IP: 91.189.88.161 80]\nFetched 101 MB in 3min 5s (547 kB/s)\nW: Duplicate sources.list entry http://archive.ubuntu.com/ubuntu/ trusty-updates/main amd64 Packages (/var/lib/apt/lists/archive.ubuntu.com_ubuntu_dists_trusty-updates_main_binary-amd64_Packages.gz)\nW: Duplicate sources.list entry http://archive.ubuntu.com/ubuntu/ trusty-updates/restricted amd64 Packages (/var/lib/apt/lists/archive.ubuntu.com_ubuntu_dists_trusty-updates_restricted_binary-amd64_Packages.gz)\nW: Duplicate sources.list entry http://archive.ubuntu.com/ubuntu/ trusty-updates/universe amd64 Packages (/var/lib/apt/lists/archive.ubuntu.com_ubuntu_dists_trusty-updates_universe_binary-amd64_Packages.gz)\nW: You may want to run apt-get update to correct these problems\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/l/linux/linux-libc-dev_3.13.0-107.154_amd64.deb  404  Not Found [IP: 91.189.88.161 80]\n\nE: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?\n</pre></div>\n\n\n<p>Looks like the docker layer that installs packages was separate from the layer that updates the package list. I'm testing a fix in <a href=\"http://build.osrfoundation.org/view/main/view/BuildCopTests/job/sdformat-ci-default-trusty-amd64/48/\" rel=\"nofollow\" class=\"ap-connect-link\">http://build.osrfoundation.org/view/main/view/BuildCopTests/job/sdformat-ci-default-trusty-amd64/48/</a></p>", "type": "rendered"}, "assignee": {"display_name": "Shane Loretz", "uuid": "{656e3311-aad9-45a1-aaf7-b0ee0e84b287}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B656e3311-aad9-45a1-aaf7-b0ee0e84b287%7D"}, "html": {"href": "https://bitbucket.org/%7B656e3311-aad9-45a1-aaf7-b0ee0e84b287%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:684383ab-ac95-4859-a350-4a6f41a94a22/c7a1ebf5-cade-4115-9f26-9d3facb776db/128"}}, "nickname": "Shane Loretz", "type": "user", "account_id": "557058:684383ab-ac95-4859-a350-4a6f41a94a22"}, "state": "resolved", "version": null, "edited_on": null, "created_on": "2017-02-06T18:16:52.964391+00:00", "milestone": null, "updated_on": "2017-02-09T17:11:49.143430+00:00", "type": "issue", "id": 100}